test/integration/029_docs_generate_tests/test_docs_generate.py

```diff
@@ -2085,8 +2110,9 @@ def expected_postgres_references_manifest(self, model_database=None):
             },
             'child_map': {
                 'model.test.ephemeral_copy': ['model.test.ephemeral_summary'],
+		        'exposure.test.notebook_exposure': ['model.test.view_summary'],
```

Comment:
ah, I just saw you have this here—I think it should go in the `parent_map` (below) instead of the `child_map` (here)